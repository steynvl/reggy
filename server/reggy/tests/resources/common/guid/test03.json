{
  "type": "GUID",
  "information": {
    "bracesAround": "Not allowed",
    "hyphensIn": "Not allowed",
    "guidCase": "Uppercase"
  },
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "commonUseCases"
}
