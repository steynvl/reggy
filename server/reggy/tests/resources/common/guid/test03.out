Java
Pattern regex = Pattern.compile("[\\dA-F]{32}");
[\\dA-F]{32}
Perl
my $regex = /[\dA-F]{32}/;
[\dA-F]{32}
POSIX
[\dA-F]{32}
[\dA-F]{32}
Python
regex = re.compile(r'[\dA-F]{32}')
[\dA-F]{32}
JavaScript
const regex = /[\dA-F]{32}/;
[\dA-F]{32}
PHP
$regex = '/[\dA-F]{32}/';
[\dA-F]{32}
Golang
regex, _ := regexp.Compile("[\\dA-F]{32}")
[\\dA-F]{32}
Rust
let re = Regex::new(r"[\dA-F]{32}").unwrap();
[\dA-F]{32}
C#
var re = new Regex(@"[\dA-F]{32}");
[\dA-F]{32}
Scala
val re = "[\\dA-F]{32}".r
[\\dA-F]{32}
Kotlin
val regex = Regex("[\\dA-F]{32}")
[\\dA-F]{32}